src/Bonus.java

```diff
@@ -1,66 +1,86 @@
 import java.util.HashSet;
 import java.util.Objects;
+import java.util.Random;
 import java.util.Scanner;
 
 public class Bonus {
 
     public static void main(String[] args) {
+        HashSet<Integer> secretNumbers = randomnumbergenerator();
+        String stringNumber = setToStringConverter(secretNumbers);
+        System.out.println(stringNumber);
+        System.out.println("Geheime code gegenereerd. Probeer te raden!");
+        feedback(stringNumber);
+    }
 
-        // Onderstaande code is uitgecommentarieerd. Haal de code uit de comment (delete /* en */) om aan de slag te gaan met de bonus
-        // In de code staan op verschillende plekken comments tussen "/*...*/" als hint wat je moet doen
-
+    public static HashSet<Integer> randomnumbergenerator(){
```

Comment:
I would use camalCase for naming of this method.
```suggestion 
    public static HashSet<Integer> randomNumberGenerator(){
```